core/src/main/scala/akka/persistence/cassandra/journal/CassandraTagRecovery.scala

```diff
@@ -59,7 +59,9 @@ trait CassandraTagRecovery {
   }
 
   private[akka] def sendTagProgress(tp: Map[Tag, TagProgress], ref: ActorRef): Future[Done] = {
+    // TODO, config or bigger?
     implicit val timeout = Timeout(1.second)
```

Comment:
right, it's a local ask without involving Cassandra, so it will only fail if system is overloaded or starved. I think a timeout of 10.s would be ok to avoid false failures